pkg/terraform/v1beta2/update.go

```diff
@@ -294,6 +295,10 @@ func updateVMwareCloudDirectorWorkerset(existingWorkerSet *kubeonev1beta2.Dynami
 		{key: "storageProfile", value: config.StorageProfile},
 		{key: "ipAllocationMode", value: config.IPAllocationMode},
 		{key: "metadata", value: config.Metadata},
+		{key: "placementPolicy", value: config.PlacementPolicy},
+		{key: "sizingPolicy", value: config.SizingPolicy},
+		{key: "diskIOPS", value: config.DiskIOPS},
+		{key: "diskBusType", value: config.DiskBusType},
```

Comment:
those new terraform variables are missed in terraform output...

Reply:
They are not required there. I just introduced them here to avoid requiring code changes in case someone wants to provide those values.

Reply:
@kron4eg modified it a bit and added comments for these fields in the terraform output.